sdk/appconfiguration/Azure.Data.AppConfiguration/Directory.Build.props

```diff
@@ -0,0 +1,9 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
```

Comment:
I assume the plan is to remove these properties once the issues are fixed in the library? If so it might be worth adding a comment to that affect and even potentially adding a link to an issue for each library.